starknet_py/net/models/transaction.py

```diff
@@ -1,3 +1,8 @@
+"""
+Dataclasses representing Transactions for internal use.
```

Comment:
These are not only for internal use, it is a public API.

Reply:
Would `library use` be better?